caculate postage/main.js

```diff
@@ -0,0 +1,20 @@
+calPostage = (postage) => {
+    var a = 25000
```

Comment:
```suggestion
    const mandatoryFee = 25000
```
không đặt tên tối nghĩa `a, b, c, ...` không được dùng `var`